Source/processcontainers/ProcessContainer.h

```diff
@@ -102,12 +102,10 @@ namespace ProcessContainers {
         // Stops the running containerized process. Returns true when stopped.
         // Note: if timeout == 0, call is asynchronous
         virtual bool Stop(const uint32_t timeout /*ms*/) = 0; 
-
-        virtual void AddRef() const = 0;
-        virtual uint32_t Release() = 0;
     };
 
-    struct IContainerAdministrator {
+    struct IContainerAdministrator 
+    {
         static IContainerAdministrator& Instance();
 
         IContainerAdministrator() = default;
```

Comment:
Perhaps better to remove this altogether for constancy if not private. We normally also not in this file declare the c'tor for an interface